lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,73 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
+        def initialize(serializer, options = {})
+          options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
+            .include_args_to_hash(options[:include])
+
+          super(serializer, options)
```

Comment:
but we only own when `new` is called, not `serializable_hash`.  So, that's why any 'config/setup' goes in the init, but everything else is triggered when the Renderer gets it (and calls to_json or whatever)